portfolio/src/main/webapp/style.css

```diff
@@ -162,10 +162,19 @@ h3 {
   float: right;
   margin-top: 4px;
   padding: 5px;
+}
+
+.down {
   transform: rotate(45deg);
   -webkit-transform: rotate(45deg);
 }
 
+.up {
+  margin-top: 8px;
+  transform: rotate(-135deg);
+  -webkit-transform: rotate(-135deg);
+}
+
```

Comment:
You might want to add comments on what these are for.

Reply:
okay